src/Modules/CalcOffence.lua

```diff
@@ -3126,13 +3126,13 @@ function calcs.offence(env, actor, activeSkill)
 					sourceHitDmg = (totalMin + totalMax) / 2 * output.PoisonDotMulti
 				end
 			end
-			if globalBreakdown then
-				globalBreakdown.PoisonDPS = {
-					s_format("Ailment mode: %s ^8(can be changed in the Configuration tab)", igniteMode == "CRIT" and "Crits Only" or "Average Damage")
-				}
-			end
 			local baseVal = calcAilmentDamage("Poison", sourceHitDmg, sourceCritDmg) * data.misc.PoisonPercentBase * output.FistOfWarAilmentEffect * globalOutput.AilmentWarcryEffect
 			if baseVal > 0 then
+				if globalBreakdown then
+					globalBreakdown.PoisonDPS = {
+						s_format("Ailment mode: %s ^8(can be changed in the Configuration tab)", igniteMode == "CRIT" and "Crits Only" or "Average Damage")
+					}
+				end
```

Comment:
Reading the code I can see that this probably doesn't harm anything, but I can't tell what it fixes by playing in PoB.  Chance to poison (and other ailments) already didn't show the poison box if you weren't doing the relevant damage

Reply:
From what I recall I made this change purely for consistency with other damage types: ``globalBreakdown.BleedDPS`` and ``globalBreakdown.IgniteDPS`` aren't populated unless their respective damage value is above zero, ``globalBreakdown.PoisonDPS`` was just a weird outlier, generating a global variable even when it was never used for anything.